Assets/HoloToolkit/Sharing/Scripts/SharingStage.cs

```diff
@@ -128,7 +135,7 @@ public string UserName
             }
             set
             {
-                using (XString userName = new XString(value))
+                using (var userName = new XString(value))
```

Comment:
Haha, I updated my formatting wizard to catch when the type is evident, like when creating new instances.  You're pretty much writing it again after declaring the type. I think it's fine as long as it's readable.